src/components/ProTip.tsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react';
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import * as React from 'react';
```

Comment:
In ts file, we still need to use * ?

Reply:
I'm not sure, I just hold my old habit. no special reason.